libs/api/domains/health-insurance-v2/src/lib/graphql/dto/AccidentStatus.input.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsNumber } from 'class-validator'
+
+@InputType()
+export class AccidentStatusInput {
```

Comment:
Should follow the [graphql naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api).